terraform/versions.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 1.5"
+  required_providers {
+    juju = {
+      source  = "juju/juju"
+      version = "~> 0.14.0"
```

Comment:
It is probably a good idea if we do not lock our TF deployments to version 0.14:

[Look at this](https://github.com/canonical/observability/pull/212/files#diff-f050dd035c535a9dc5d61f4514cb38df2338426605c532ef89573aaf77f14195)